app/templates/gulpfile.js

```diff
@@ -294,41 +436,49 @@ gulp.task('optimize', ['inject', 'test'], function() {
   log('Optimizing the js, css, and html');
 
   // Filters are named for the gulp-useref path
-  var cssFilter = $.filter('**/*.css', {restore: true});
-  var jsAppFilter = $.filter('**/' + config.optimized.app, {restore: true});
-  var jslibFilter = $.filter('**/' + config.optimized.lib, {restore: true});
+  var cssFilter = $.filter('**/*.css', {
+    restore: true
+  });
+  var jsAppFilter = $.filter('**/' + config.optimized.app, {
+    restore: true
+  });
+  var jslibFilter = $.filter('**/' + config.optimized.lib, {
+    restore: true
+  });
```

Comment:
Can you put the `{ restore: true }` in a var, so that this will look more dense?